src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,8 @@
+$page-color: #b0c4de;
+$animation-hour-time: 86400s;
+$animation-minute-time: 720s;
```

Comment:
The minute hand makes a full rotation in 60 minutes, which equals to 3600s

Reply:
Oh...such a dumb mistakes, sorry